deps/Makefile

```diff
@@ -740,6 +732,9 @@ PCRE_OBJ_TARGET = $(build_shlibdir)/libpcre2-8.$(SHLIB_EXT)
 
 # Force optimization for PCRE flags (Issue #11668)
 PCRE_CFLAGS = -O3
+ifneq ($(OS),WINNT)
+PCRE_LDFLAGS = "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
Hmmm.  It looks like we don't mess with `rpaths` inside of `make install` anymore?  How does that work?  We rely on the `rpath` entries built inside of `julia` itself?